src/main/java/edu/harvard/iq/dataverse/SettingsWrapper.java

```diff
@@ -150,6 +150,10 @@ public boolean isRsyncDownload() {
         return systemConfig.isRsyncDownload();
     }
     
+    public boolean isHTTPUpload(){
```

Comment:
Should you use the `SettingsWrapper` cache for this?

Reply:
That bit of code is from dual mode. I closed this pr because the comparison isn't accurate.